virtualization.m

```diff
@@ -357,16 +357,12 @@ void setStorageDevicesVZVirtualMachineConfiguration(void *config,
  */
 void *newVZDiskImageStorageDeviceAttachment(const char *diskPath, bool readOnly, void **error)
 {
-    VZDiskImageStorageDeviceAttachment *ret;
-    @autoreleasepool {
-        NSString *diskPathNSString = [NSString stringWithUTF8String:diskPath];
-        NSURL *diskURL = [NSURL fileURLWithPath:diskPathNSString];
```

Comment:
These codes do not use `alloc` or `init`, `new` to create new objects, so there was no need to release them.